test/data.js

```diff
@@ -1,5 +1,21 @@
-var ActionTypes = require('butter-action-types')
-var QRCode = require('butter-component-qrcode').default
+var React = require('react');
+var ActionTypes = require('butter-action-types');
+var QRCode = require('butter-component-qrcode').default;
+
+//Modal content test
+var ModalContent = React.createClass({
+  render: function() {
+    return React.createElement("img", {src: "https://media.giphy.com/media/o0vwzuFwCGAFO/giphy.gif"});
+  }
+});
+
+//QR component opts
+var QR_opts = {
+    ipPddress: '192.168.0.1',
+    httpApiPort: '8080',
+    httpApiUsername: 'testuser',
+    httpApiPassword: 'testpassword'
+};
```

Comment:
Like this? https://github.com/buttercomponents/butter-component-settings/pull/52/commits/d594a5da60d6fbd65d5b47eba4e323c26c89159b